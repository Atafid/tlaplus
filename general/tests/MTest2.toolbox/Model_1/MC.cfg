SPECIFICATION spec_124950962213710000
\* Generated at Wed Aug 05 15:00:22 PDT 2009