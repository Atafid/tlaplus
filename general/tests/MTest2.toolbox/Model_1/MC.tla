---- MODULE MC ----
EXTENDS MTest2, TLC

\* Specification formula
spec_124950962213710000 ==
(x=0)
----
====
\* Generated at Wed Aug 05 15:00:22 PDT 2009