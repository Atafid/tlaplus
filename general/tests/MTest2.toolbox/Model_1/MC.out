TLC2 Version 2.02 of 3 August 2009
@!@!@STARTMSG 2187:0 @!@!@
Running in Model-Checking mode.
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file MC.tla
Parsing file MTest2.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\TLC.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Naturals.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Sequences.tla
Semantic processing of module MTest2
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module TLC
Semantic processing of module MC
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting...
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
@!@!@STARTMSG 2190:0 @!@!@
Finished computing initial states: 1 distinct state generated.
@!@!@ENDMSG 2190 @!@!@
@!@!@STARTMSG 2193:0 @!@!@
Model checking completed. No error has been found.
  Estimates of the probability that TLC did not check all reachable states
  because two distinct states had the same fingerprint:
  calculated (optimistic):  val = 0
  based on the actual fingerprints:  val = 1.1E-19
@!@!@ENDMSG 2193 @!@!@
@!@!@STARTMSG 2201:0 @!@!@
The coverage statistics :
@!@!@ENDMSG 2201 @!@!@
Exception in thread "main" java.lang.NullPointerException
	at java.lang.AbstractStringBuilder.replace(AbstractStringBuilder.java:805)
	at java.lang.StringBuffer.replace(StringBuffer.java:391)
	at tlc2.output.MP.replaceString(MP.java:996)
	at tlc2.output.MP.getMessage(MP.java:744)
	at tlc2.output.MP.printError(MP.java:873)
	at tlc2.output.MP.printError(MP.java:860)
	at tlc2.output.MP.printError(MP.java:885)
	at tlc2.TLC.process(TLC.java:550)
	at tlc2.TLC.main(TLC.java:142)
